tests/op/test_model_function.py

```diff
@@ -73,3 +69,21 @@ def test_model_output_str(trust_remote_code, model_name="microsoft/deberta-v3-ba
             and out["translation"][1] == data[1][0]
             and out["translation"][2] == data[2][0]
         )
+
+
+def test_output_dict(model_name="microsoft/deberta-v3-base"):
```

Comment:
add one for string just to ensure the meta thingy? and maybe one where one of the value types is string while the other is int?

e.g. `{"a" : [1,2,3], "b" : ["str1", "str2", "str3"]}` and I believe this should error or be upcasted

Reply:
Let me look into this case and how to support this behavior cleanly. Maybe we expand this to support `meta` and passing in a dict instead of `output_columns`

Reply:
This was completed